app/pages/resource/reservation-form/ReservationFormContainer.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+
+import ReservationForm from './ReservationForm';
+
+const resourceSelector = (state, props) => props.resource;
```

Comment:
- [ ] At the moment we have all container selectors defined in separate files instead of in the container file. The main reason is that each file should have one clear purpose and not do multiple things. In a simple case like this I think it's okay to have the selector in the container file, but for consistency, it would then be better to do it the same way in all containers. And in that case I think it would be better to have a hard rule that container files should not have ANY render logic and only connect a component. Maybe that would be a good rule in any case. That way the container file would still have only one purpose even if the selector is defined in it.

So regarding containers and their selectors, I think the good options are:
1. Define selectors in containers, don't render anything in a container.
2. Define selectors in a separate file but still don't render anything in a container.
3. Keep using the current style where selectors are defined in a separate file, and containers can have rendering logic in them.

Some thoughts:
- With longer or more complicated selectors, it's nice to have them in a separate file.
- We define actions (mapped with dispatch) in the container so in that regard why not have the selectors in the container file as well. Or would it be better to have the actions in a separate file as well and define the actions needed in container and the connected component there.
- Some containers will need componentDidMount etc. so they will have some logic outside of connecting.

So I guess the question is what to have in container files and what not to have there (selectors, actions, React lifecycle methods, rendering).

Thoughts @FelixLoether and @YoYuUm.